src/design/inputs/multi-select.js

```diff
@@ -123,5 +123,6 @@ const SelectEl = styled.div`
 
 const MultiSelectEl = styled.div`
   min-width: 280px;
-  max-width: 360px;
+  max-width: 550px;
```

Comment:
How does this look on phone screens that are less than 550px wide?

Reply:
Discussed.